src/lib/pages/frontsite/SelfCheckout/SelfCheckout.js

```diff
@@ -41,50 +38,10 @@ class SelfCheckout extends React.Component {
       this.setState({ barcode: detectedBarcode });
       const { selfCheckOutSearch } = this.props;
       await selfCheckOutSearch(detectedBarcode);
-
-      // open modal if item is loanable
-      const shouldShowModal = this.isItemLoanable(detectedBarcode);
-      if (shouldShowModal) {
-        this.toggleModal(true);
-      } else {
-        this.toggleModal(false);
-      }
+      this.toggleModal(true);
     }
   };
 
-  itemStatus = (item) => ({
```

Comment:
looks like a lot of the code deleted here, just to understand, what was it replaced by?

Reply:
Errors are handled on the backend now. See here: https://github.com/inveniosoftware/invenio-app-ils/pull/1223